science/gmt5/Portfile

```diff
@@ -41,15 +40,16 @@ if {${subport} eq "gmt5"} {
                         sha256  078d4997507cb15344c74a874568985e45bdbd6d3a72d330c74c47f4c0359bb1 \
                         size    59175704
 } else {
-    checksums           rmd160  2392d55baadcf02b5c5766fa95264a039a248a8f \
-                        sha256  176f19b2890db52cd811820ca87a3c27149d5a4dde239d49c2f0da34c218b61a \
-                        size    66926028
+    checksums           rmd160  a53d2bd50a960dd31893ca377d36a188c5e07f88 \
+                        sha256  8b91af18775a90968cdf369b659c289ded5b6cb2719c8c58294499ba2799b650 \
+                        size    66892468
 }
 
 depends_lib         port:dcw-gmt \
                     port:ghostscript \
                     port:gshhg-gmt \
-                    port:netcdf
+                    port:netcdf \
+                    port:curl
```

Comment:
Please add this new dependency in alphabetical order. So put it at the beginning of the list.